rest/taskana-rest-spring/pom.xml

```diff
@@ -1,196 +1,201 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
Can you please create a ticket that we still have to find a solution for our formatter in xml?

Reply:
This is the second time we get reminded :)